app/components/billing-information.js

```diff
@@ -0,0 +1,16 @@
+import Component from '@ember/component';
+import { countries } from '../utils/countries';
+import { equal }  from '@ember/object/computed';
+
+const SCROLL = {
+  CONTACT: 'contactDetails',
+  BILLING: 'billingDetails',
+};
+
+export default Component.extend({
+  countries,
+  scrollSection: null,
+
+  scrollToContactDetails: equal('scrollSection', SCROLL.CONTACT),
+  scrollToBillingDetails: equal('scrollSection', SCROLL.BILLING)
+});
```

Comment:
Eventually I'd suggest to move all `billing-*` components to `billing` subfolder: `components/billing/*.js`. It's too many of them and they are flooding the root components folder.